src/Numerical/RootFinding/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(MY_LIBRARY_SOURCES
+    RootFinding/QuadraticEquation.cpp)
```

Comment:
Moving the closing paren to the next line may reduce future conflicts (this shamelessly stolen from @nilsdeppe).